src/components/app/footer.tsx

```diff
@@ -186,9 +186,9 @@ const DarkModeToggle = () => {
   }
   return (
     <div className="flex items-center justify-between">
-      <h2 className="hidden mr-3 sm:block">
+      <span className="hidden mr-3 sm:block">
         {resolvedTheme === 'dark' ? 'Dark' : 'Light'} Mode
-      </h2>
+      </span>
```

Comment:
darkmode toggle isn't important for the page content so doesn't need to be an `h2`